projects/aca-shared/src/lib/components/tool-bar/toolbar-menu/toolbar-menu.component.html

```diff
@@ -12,14 +12,19 @@
 </button>
 
 <mat-menu #menu="matMenu" [overlapTrigger]="false">
-  <ng-container *ngFor="let child of actionRef.children; trackBy: trackByActionId">
-    <ng-container [ngSwitch]="child.type">
+  <ng-container *ngFor="let child of actionRef.children; trackBy: trackByActionId" [ngSwitch]="child.type">
       <ng-container *ngSwitchCase="'custom'">
-        <adf-dynamic-component [id]="child.component" [data]="child.data"></adf-dynamic-component>
+        <button mat-menu-item style="padding: 0" (click)="onCustomItemContainerClick($event)">
```

Comment:
don't embed styles in the components, we have css/scss for that

Reply:
Agreed. I'll move the styles to the component.